 

public class Launch19   {

	public static void main(String[] args)  {
		// TODO Auto-generated method stub
System.out.println("shiva");
try {
// int arr[]=new int[99999999999]; this is out of memory error not an exception 
//exception can be handled by try catch block but error cannot be handled by try catch block   

}
catch(Exception e)
{
	System.out.println("shiva");
}
	}

}
